src/main/java/org/spongepowered/api/GameRegistry.java

```diff
@@ -396,6 +411,185 @@
     Collection<String> getDefaultGameRules();
 
     /**
+     * Gets the {@link Statistic} with the specified id.
+     *
+     * @param id The id of the statistic to return
+     * @return The statistic with the given id or Optional.absent() if not found
+     */
+    Optional<? extends Statistic> getStatistic(String id);
```

Comment:
Overwriteability
That are probably remainings of my PR that uses this kind of generics everywhere in the GameRegistry.
Shall i remove it?